spec/models/repository_spec.rb

```diff
@@ -18,56 +18,59 @@
 
   describe 'scopes' do
     describe '.only_mirrored' do
-      it 'returns only mirrored repos' do
-        mirrored = create :repository, mirroring_enabled: true
-        create :repository, mirroring_enabled: false
+      subject { Repository.only_mirrored }
```

Comment:
`Repository` -> `described_class` :grimacing:

Reply:
:man_facepalming:

Reply:
Rubocop didn't catch that, weird.

Reply:
Yeah, I though we had a rule for that.